packages/web-components/fast-foundation/src/avatar/avatar.template.ts

```diff
@@ -0,0 +1,31 @@
+import { html, when } from "@microsoft/fast-element";
+import { Avatar } from "./avatar";
+
+/**
+ * The template for {@link @microsoft/fast-foundation#Avatar} component.
+ * @public
+ */
+export const AvatarTemplate = html<Avatar>`
+  <template>
+    <div 
+      class="coin ${x => (x.shape === "square" ? "square" : "circle")}"
+      style="${x =>
+        x.fill || x.color
+            ? `background-color: var(--badge-fill-${x.fill}); color: var(--badge-color-${x.color})`
```

Comment:
Should this be `badge-fill-____` and `badge-color-____` or should we go with a more semantic name for mapping here?

Reply:
That, my friend, is a copy-pasta mistake. Will fix, thanks.